apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx

```diff
@@ -108,7 +116,7 @@ export default function useAssetActions({ currency, accounts }: useAssetActionsP
                   },
                 },
               ] as const,
-              disabled: areAccountsBalanceEmpty,
+              disabled: isPtxServiceCtaScreensDisabled || areAccountsBalanceEmpty,
```

Comment:
this memoized value is now missing the `isPtxServiceCtaScreensDisabled` dep (but could perhaps be declared inside if you're trying to avoid a memoized dependency in another memoized value?)

Reply:
Thank you, updated to declare inside the existing useMemo 👍